Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:2
```

Comment:
First of all thank you very much for your contribution! It is great to see that someone is finding this script helpful.

Secondly, would it be possible to use an alpine python image instead? It would be much better if the image is as small as posisble

Reply:
Sure it can be done.
The only thing to weigh in is if the Alpine base image has no disadvantages for the dev/test usage case. Apart from the Automated Build being a distribution method, the Dockerfile serves the local dev/test workflow. I don't know about Alpine image fitness to dev.
What do you think?